llvm/lib/CodeGen/Yk/FixStackmapsSpillReloads.cpp

```diff
@@ -190,6 +190,12 @@ bool FixStackmapsSpillReloads::runOnMachineFunction(MachineFunction &MF) {
           }
           MOI++;
         }
+        MachineOperand *Last = NewMI->operands_end() - 1;
```

Comment:
Is the `-1` always safe? Can there ever be empty operands or whatnot?

Reply:
I'm not quite sure I understand the relationship between `MIB` and `NewMI`. I'm very happy to believe it's OK but just want to be sure you've checked :)

Reply:
OK.